### ISSUE TITLE
Does SDPi need to define a SDC suitable slewing period ?

### ISSUE THREAD

**Issue by PaulMartinsen** [IHE/DEV.SDPi#295]
Some sources indicate the slewing period, the amount of time a device will take to bring its internal clock in-line with a time-stamp supplied by an NTP source, could be [300s](https://www.eecis.udel.edu/~mills/ntp/html/clock.html#step) or [600s](https://stackoverflow.com/questions/33746324/ntp-questions-slewing-versus-jumping-the-time). Are system defaults for NTP synchronization acceptable for SDC?

**Comment by PaulMartinsen** [comment]
The [`DATE`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date) header provides a well-known mechanism for SDC participants to communicate their current time in every message. [The date field is mandatory](https://httpwg.org/specs/rfc9110.html#field.date) for origin servers with a clock.

**Comment by PaulMartinsen** [comment]
This may be resolved by [PR354](https://github.com/IHE/DEV.SDPi/pull/354), which suggests consumers check the time in subscription response messages, for example (R1601)

**Comment by JavierEspina** [comment]
I agree that https://github.com/IHE/DEV.SDPi/pull/354 (which is incorporated in SDPi 2.1) most likely resolves this issue.

**Comment by JavierEspina** [comment]
**SDPi call 20-Jun-2025:**

Decided to close this issue as it is no longer needed, as discussed above.
